docker/docker-compose.tests.yml

```diff
@@ -9,6 +9,8 @@ services:
     build:
       context: ../
       dockerfile: docker/Dockerfile.test-database
+    environment:
+      MB_IMPORT_DUMPS: "false"
```

Comment:
Yeah, I missed that you didn't have the brackets before. Thanks for changing it though. 👍